{"name": "Scenario name: Verificar que un usuario puede iniciar sesión correctamente con credenciales válidas", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'page'\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"steps/login_steps.py\", line 9, in step_impl\n    context.login_page = LoginPage(context.page)\n                                   ^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "Given que estoy en la página de inicio de sesión", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'page'\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"steps/login_steps.py\", line 9, in step_impl\n    context.login_page = LoginPage(context.page)\n                                   ^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "start": 1751742383013, "stop": 1751742383014}, {"name": "When introduzco un nombre de usuario y contraseña válidos", "status": "skipped", "start": 1751742383014, "stop": 1751742383014}, {"name": "And hago clic en el botón de Iniciar Sesión", "status": "skipped", "start": 1751742383014, "stop": 1751742383014}, {"name": "Then debería ser redirigido al tablero con un mensaje de bienvenida \"You logged into a secure area!\"", "status": "skipped", "start": 1751742383014, "stop": 1751742383014}, {"name": "And cierro sesión", "status": "skipped", "start": 1751742383014, "stop": 1751742383014}], "start": 1751742383012, "stop": 1751742383014, "uuid": "63d458e0-f4bb-4c4b-8953-fb1f360143d1", "historyId": "2d5c843281384bd00a4d45173783ed41", "fullName": "Funcionalidad de Inicio de Sesión de Usuario: Scenario name: Verificar que un usuario puede iniciar sesión correctamente con credenciales válidas", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "login_valid"}, {"name": "feature", "value": "Funcionalidad de Inicio de Sesión de Usuario"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}